docs/reference/integrations.md

```diff
@@ -0,0 +1,27 @@
+# Integrations
+
+### agent
+
+_Interface_: jenkins*agent_v0  
+\_Supported charms*: [jenkins-agent-k8s](https://charmhub.io/jenkins-agent-k8s),
+[jenkins-agent](https://charmhub.io/jenkins-agent)
```

Comment:
This looks weird when rendered, is it as intended?

Reply:
My markdown autoformatter on save did that! Thanks for catching that!